src/lxc/cgroups/cgfsng.c

```diff
@@ -262,6 +262,50 @@ static int lxc_cpumask(char *buf, uint32_t **bitarr, size_t *last_set_bit)
 	return 0;
 }
 
+static int lxc_cpumask_update(char *buf, uint32_t *bitarr, size_t last_set_bit,
+			      bool clear)
+{
+	bool flipped = false;
+	char *token;
+
+	lxc_iterate_parts(token, buf, ",") {
+		unsigned last_bit, first_bit;
+		char *range;
+
+		errno = 0;
+		first_bit = strtoul(token, NULL, 0);
+		last_bit = first_bit;
+		range = strchr(token, '-');
+		if (range)
+			last_bit = strtoul(range + 1, NULL, 0);
+
+		if (!(first_bit <= last_bit)) {
+			WARN("The cup range seems to be inverted: %u-%u", first_bit, last_bit);
```

Comment:
Minor comment: the word "cup" should be "cpu".

Reply:
Yup, thanks.